apps/SageAccountWeb/src/components/ConfigureEmail.tsx

```diff
@@ -69,6 +69,7 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
           accessToken,
         ).then(() => {
           displayToast('Email has been successfully added', 'success')
+          refetchCurrentProfile()
```

Comment:
PORTALS-3101: Refetch after new email has been successfully added to the account